### PR TITLE
Array_concat process

### PR DESCRIPTION
**Pull request by zansinergise**
https://git.sinergise.com/team-6/openeo-platform/-/issues/62

### CODE REVIEW

**Review by dgostencnik — CHANGES_REQUESTED** [review]
similar as in  https://github.com/openEOPlatform/openeo-pg-evalscript-converter/pull/15 check mandatory inputs

**Comment by zansinergise** [comment]
> similar as in #15 check mandatory inputs

Fixed the process accordingly and added additional test cases

File: src/pg_to_evalscript/javascript_processes/array_concat.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function array_concat(arguments) {
+  const { array1, array2 } = arguments;
+  if (array1 === null || array1 === undefined) {
+    throw new Error(
+      "Mandatory argument `array1` is either null or not defined."
+    );
+  }
+  if (array2 === null || array2 === undefined) {
```
Comment (dgostencnik) [inline src/pg_to_evalscript/javascript_processes/array_concat.js:8]: Perhaps it would also make sense to check if both arguments are arrays. fftr

**Review by dgostencnik — APPROVED** [review]

File: src/pg_to_evalscript/javascript_processes/array_concat.js (hunk 1 above)
Comment (zansinergise) [reply src/pg_to_evalscript/javascript_processes/array_concat.js:8]: Yeah that makes sense. Added and will also add for #15

Comment (dgostencnik) [reply src/pg_to_evalscript/javascript_processes/array_concat.js:8]: :+1:

**Pull request merged**